gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Jan 18 11:31:25 CET 2018
+#Fri Jan 19 09:36:13 CET 2018
```

Comment:
isn't that generating a bit of noise - especially since we get this from our scm

Reply:
We could remove gradle-wrapper from the repo, the idea is that you can build it without a local gradle install.

Reply:
Done, removed the first line with the date. Gradle also respects this and does not add it back.